### DISCUSSION TITLE
Disable friendly snippets

### DISCUSSION THREAD

**Discussion by joshstobbs** [VonHeikemen/lsp-zero.nvim#48] [General]
Hey there!  Just wondering if there is a way to disable the included snippets?  I'd love to keep the snippet engine but not populate it with `friendly-snippets`.

Thanks so much!

**Comment by VonHeikemen** [comment]
Uninstall them. Everything should still work.

**Comment by emilBeBri** [comment]
Could you tell me how you did that OP? Would like to learn, and put in my own snippets in the engine as well.
